docs/api.rst

```diff
@@ -10469,7 +10469,7 @@ Querying general information
 	      "backend_default_arguments": {
 	          "max-logfiles-num": 3,
 	          "max-size-in-mb-all-logs": 300,
-	          "sqlite-instructions": 5000,
+	          "sqlite-instructions": 5000
 	      },
```

Comment:
This `,` is also extra

Reply:
what?

Reply:
I ran it through a linter. It had also more stuff. It should be okay now.